### PR TITLE
Ember 3.6 update

### PR DESCRIPTION
**Pull request by martijnbolhuis**
This is an attempt to upgrade to ember 3.6.1 using ember-cli-update. This upgrade seems to work when used in my own app. However, still one of the tests in this add-on seems to fail. This test is related to _belongs-to-sticky_. Could you have a look?

```
not ok 7 Chrome 71.0 - [6 ms] - integration/query-has-many: Querying has-many relationship multiple times doesn't clear belongs-to-sticky association                                                                                         
    ---
        actual: >
            null
        stack: >
            TypeError: computed is not a function
                at RelationshipQuery.belongsToSticky (http://localhost:7357/assets/vendor.js:51118:12)
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:226:46)
                at runTest (http://localhost:7357/assets/test-support.js:4178:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4164:6)
                at http://localhost:7357/assets/test-support.js:4385:12
                at advanceTaskQueue (http://localhost:7357/assets/test-support.js:3777:6)
                at advance (http://localhost:7357/assets/test-support.js:3758:4)
                at advanceTestQueue (http://localhost:7357/assets/test-support.js:3803:4)
                at Object.advance (http://localhost:7357/assets/test-support.js:3761:5)
                at begin (http://localhost:7357/assets/test-support.js:5530:20)
        message: >
            Died on test #1     at Object.<anonymous> (http://localhost:7357/assets/tests.js:224:21)
                at processModule (http://localhost:7357/assets/test-support.js:3544:16)
                at module$1 (http://localhost:7357/assets/test-support.js:3569:4)
                at Module.callback (http://localhost:7357/assets/tests.js:171:21)
                at Module.exports (http://localhost:7357/assets/vendor.js:111:32)
                at requireModule (http://localhost:7357/assets/vendor.js:32:18)
                at TestLoader.require (http://localhost:7357/assets/test-support.js:11295:9): computed is not a function
        negative: >
            false
        Log: |
```

### CODE REVIEW

File: addon/belongs-to-sticky.js [hunk 1]
```diff
@@ -17,13 +16,13 @@ var recordHasId = function (record) {
...
  */
-var belongsToSticky = function () {
-  var computed = DS.belongsTo(...arguments);
-  var meta = computed.meta();
+const belongsToSticky = function () {
+  let computed = DS.belongsTo(...arguments);
+  let meta = computed.meta();
   meta.sticky = true;
-  return Ember.computed({
+  return computed({
```
Comment (yads) [inline addon/belongs-to-sticky.js]: This doesn't seem right, it's supposed to use Ember.computed, but is using the computed variable fromthe DS.belongsTo call a few lines above.

Comment (RubenSmit) [reply addon/belongs-to-sticky.js]: Ah, i suppose that might be te problem I ran into, will try to fix it.

Comment (RubenSmit) [reply addon/belongs-to-sticky.js]: So, I have fixed the problem, and prefixed the computed from DS.belongsTo with an underscore, and it is all working now.

**Comment by RubenSmit** [comment]
All tests are passing now, and the ember version has been bumped to 3.7.

File: ember-modules-codemod.tmp.17095 [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+[2,"_RegistryProxyMixin",6,"tests/helpers/owner.js","\nif (Ember._RegistryProxyMixin && Ember._ContainerProxyMixin) {\n  Owner = Ember.Object.extend(Ember._RegistryProxyMixin, Ember._ContainerProxyMixin);\n} else {\n  Owner = Ember.Object.extend();"]
+[2,"_ContainerProxyMixin",6,"tests/helpers/owner.js","\nif (Ember._RegistryProxyMixin && Ember._ContainerProxyMixin) {\n  Owner = Ember.Object.extend(Ember._RegistryProxyMixin, Ember._ContainerProxyMixin);\n} else {\n  Owner = Ember.Object.extend();"]
+[2,"Registry",10,"tests/helpers/store.js","  options = options || {};\n\n  if (Ember.Registry) {\n    registry = env.registry = new Ember.Registry();\n    owner = Owner.create({"]
+[2,"Registry",11,"tests/helpers/store.js","\n  if (Ember.Registry) {\n    registry = env.registry = new Ember.Registry();\n    owner = Owner.create({\n      __registry__: registry"]
+[2,"Container",20,"tests/helpers/store.js","    owner.__container__ = container;\n  } else {\n    container = env.container = new Ember.Container();\n    registry = env.registry = container;\n  }"]
```
Comment (mdehoog) [inline ember-modules-codemod.tmp.17095]: should this be ignored/removed?

**Comment by mdehoog** [comment]
thank you for doing this, looks great!

File: ember-modules-codemod.tmp.17095 (hunk 2 above)
Comment (RubenSmit) [reply ember-modules-codemod.tmp.17095]: Whoops, its a temp file that should have been removed automatically.

**Comment by RubenSmit** [comment]
I removed the mistakenly committed temp file.

**Comment by ELLIOTTCABLE** [comment]
Great news!

Please mint a version as soon as possible, so we can deprecate our fork? Pretty, pretty please? 👐 😆

**Comment by martijnbolhuis** [comment]
Could you please publish this upgrade to npm?

**Comment by mdehoog** [comment]
Apologies for the delay! I just published `v0.3.0`.

**Pull request merged**